signicat/Issues/kvkTradeRegister/description.xml

```diff
@@ -455,8 +458,8 @@
 				<nl>Peilmoment (opvraagmoment uit handelsregister)</nl>
 			</Name>
 			<Description>
-				<en>Reference moment is the moment the data is extracted from the trade register</en>
-				<nl>Peilmoment is het moment dat de gegevens uit het handelsregister zijn onttrokken</nl>
+				<en>Reference moment is the moment the data is extracted from the trade register (format yyyymmddhhmmssmmm)</en>
+				<nl>Peilmoment is het moment dat de gegevens uit het handelsregister zijn onttrokken (formaat jjjjmmddhhmmssmmm)</nl>
```

Comment:
it seems that `to extract` and `ontrekken` in general mean: _to remove_ which seems not entirely match the meaning of _reference_ or _opvraagmoment_